test/integration/other/form-entities-version.js

```diff
@@ -270,15 +285,18 @@ describe('Update / migrate entities-version within form', () => {
       // Upload an attachment
       await asAlice.post('/v1/projects/1/forms/withAttachments/draft/attachments/goodone.csv')
         .send('test,csv\n1,2')
-        .set('Content-Type', 'text/csv');
+        .set('Content-Type', 'text/csv')
+        .expect(200);
 
       // Publish the draft
-      await asAlice.post('/v1/projects/1/forms/withAttachments/draft/pubilsh');
+      await asAlice.post('/v1/projects/1/forms/withAttachments/draft/publish')
+        .expect(200);
 
       // Create a draft
-      await asAlice.post('/v1/projects/1/forms/withAttachments/draft');
+      await asAlice.post('/v1/projects/1/forms/withAttachments/draft')
+        .expect(200);
 
-      await asAlice.get('/v1/projects/1/forms/withAttachments/attachments')
+      await asAlice.get('/v1/projects/1/forms/withAttachments/draft/attachments')
```

Comment:
I didn't see a typo above but i see the one here about getting the draft attachments instead of the published attachments. Just wanted to say I approve of this change here.

Reply:
Above it was `pubilsh` instead of `publish`!